webpack/components/QuestionForm.js

```diff
@@ -303,6 +317,11 @@ class QuestionForm extends Component{
     });
   }
 
+  actionWord() {
```

Comment:
Should you just change this to 'Save' per our user feedback discussion yesterday afternoon?